### PR TITLE
[CPDP-1252] - Test bug fix

### PR DESCRIPTION
**Pull request by modivilioglu**

### CODE REVIEW

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/ytdliability/estimated_income.scala.html [hunk 1]
```diff
@@ -69,7 +69,6 @@
 
         @details(messages(s"$key.details.summary"),
             Html(s"""|
-            |<p>${messages(s"$key.details.p1", taxYearStartYear, taxYearEndYear)}</p>
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/ytdliability/estimated_income.scala.html]: Do we not need p1 anymore? We can delete this from the messages file then to clean it up if we don't. Also then it'll look nicer if p2 became p1 in the key if you delete p1.

Comment (modivilioglu) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/ytdliability/estimated_income.scala.html]: Sure.

Comment (modivilioglu) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/ytdliability/estimated_income.scala.html]: Done.

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**